pkg/catalog/catalog.go

```diff
@@ -146,7 +147,7 @@ func SyncTemplates(ctx context.Context, mc model.ClientSet, c *model.Catalog) er
 				repo.Driver = c.Type
 
 				t := &model.Template{
-					Name:        repo.Name,
+					Name:        strings.TrimPrefix(repo.Name, "terraform-"),
```

Comment:
Let's use a function like `normalizeTemplateName` to make the code self-descriptive.